hw2/03-form.js

```diff
@@ -1,3 +1,53 @@
-/** Exercise 03 - Form **/
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-console */
+/** Exercise 03 - Form * */
 
 // Add your code here
+
+// Function to get data by the dom and print to the console
+const onSubmit = () => {
+  const user = document.getElementById('nameInput').value;
+  const email = document.getElementById('emailInput').value;
+  const text = document.getElementById('textInput').value;
+  const newsLet = document.getElementById('newsLetterInput').checked;
+  const but = document.getElementById('sub');
+  let toPrint = '';
+  let feedback = '';
+
+  if (newsLet) {
```

Comment:
The handling of these console log messages is so smart! I don't know why I didn't think about putting the ones with conditions into their own variable... mine looks like a mess lol. Love this idea!